app/src/main/java/com/developer/abhinavraj/moviesearch/MainActivity.java

```diff
@@ -1,13 +1,30 @@
 package com.developer.abhinavraj.moviesearch;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.widget.ListView;
 
 public class MainActivity extends AppCompatActivity {
 
+    ListView listView;
+    String[] nameArray = {"movie1", "movie2"};
+    String[] infoArray = {"date1", "date2"};
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Toolbar toolbar = findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        CustomListAdapter cs = new CustomListAdapter(this, nameArray, infoArray);
+        listView = (ListView) findViewById(R.id.listviewID);
+        listView.setAdapter(cs);
+
```

Comment:
Why this extra line, remove it!

Reply:
You don't have to make the adapter, your task was just to make a simple xml view